setup.py

```diff
@@ -35,10 +41,6 @@ def get_install_requires():
             continue
         # add line to requirements
         requirements.append(line.replace('\n', ''))
-    # add py2-ipaddress if python2
```

Comment:
you can test it by generating the packages with:

```
python setup.py sdist bdist_wheel
```

Then create 2 new virtualenvs, one with python2 and one with python3, and in each virtualenv you do:

```
pip install -U pip wheel setuptools
pip install -r requirements-test.txt
```

Then, for each virtualenv you do:

```
cd dist/
pip install ./<name-of-tar.gz-package>
cd ..
./runtests.py
```

I did the same also for the wheel package (uninstalling the dependencies and reinstalling the package).

Without this test we would risk to release a broken package and would piss many people off!